packages/@sanity/form-builder/src/inputs/PortableText/object/BlockObjectPreview.tsx

```diff
@@ -22,8 +37,11 @@ const POPOVER_PROPS: MenuButtonProps['popover'] = {
 }
 
 export function BlockObjectPreview(props: BlockObjectPreviewProps) {
-  const {value, type, readOnly, onClickingEdit, onClickingDelete} = props
+  const {focused, value, type, readOnly, onClickingEdit, onClickingDelete} = props
+  const editor = usePortableTextEditor()
   const menuButtonId = useId()
+  const menuButton = useRef<HTMLButtonElement>()
+  const isTabbing = useRef<boolean>(false)
   const isCustomPreviewComponent = Boolean(type.preview?.component)
   const layout = 'block'
```

Comment:
This constant could be moved to the top level of the component: `const LAYOUT = 'block'`

Reply:
Yeah, agree!